src/popover/popover.directive.ts

```diff
@@ -54,10 +66,11 @@ export class PopoverDirective implements OnInit, OnDestroy {
   @Output() public onHidden: EventEmitter<any>;
 
   private _popover: ComponentLoader<PopoverContainerComponent>;
+  private _outsideClickListener: Function;
 
-  public constructor(_elementRef: ElementRef,
-                     _renderer: Renderer,
-                     _viewContainerRef: ViewContainerRef,
+  public constructor(protected _elementRef: ElementRef,
```

Comment:
yep, it is. I am guilty, but I prefer implicit approach over explicit
in this case I don't what to give this binding to TS compiler